js/main.js

```diff
@@ -243,12 +243,12 @@ $(document).ready(function(){
         var seconds = Math.floor((distance % (1000 * 60)) / 1000);
```

Comment:
Update the timer to show doubly digit figures by :
`var formattedNumber = ("0" + myNumber).slice(-2);`

Reply:
@pvgupta24 done in the next commit